test/Graphics/Transformable.cpp

```diff
@@ -0,0 +1,123 @@
+#include <SFML/Graphics/Transformable.hpp>
+#include "GraphicsUtil.hpp"
+
+#include <doctest.h>
+
+using doctest::Approx;
+
+TEST_CASE("sf::Transformable class - [graphics]")
+{
+    SUBCASE("Construction")
+    {
+        const sf::Transformable transformable;
+        CHECK(transformable.getPosition() == sf::Vector2f(0, 0));
+        CHECK(transformable.getRotation() == 0);
+        CHECK(transformable.getScale() == sf::Vector2f(1, 1));
+        CHECK(transformable.getOrigin() == sf::Vector2f(0, 0));
+        CHECK(transformable.getTransform() == sf::Transform());
+        CHECK(transformable.getInverseTransform() == sf::Transform());
+    }
+
+    SUBCASE("Setters and getters")
+    {
+        sf::Transformable transformable;
+
+        CHECK(transformable.getPosition() == sf::Vector2f(0, 0));
+        transformable.setPosition({3, 4});
+        CHECK(transformable.getPosition() == sf::Vector2f(3, 4));
+
+        CHECK(transformable.getRotation() == 0);
+        transformable.setRotation(3.14f);
+        CHECK(transformable.getRotation() == 3.14f);
+        transformable.setRotation(540);
+        CHECK(transformable.getRotation() == 180);
+        transformable.setRotation(-72);
+        CHECK(transformable.getRotation() == 288);
+
+        CHECK(transformable.getScale() == sf::Vector2f(1, 1));
+        transformable.setScale({5, 6});
+        CHECK(transformable.getScale() == sf::Vector2f(5, 6));
+
+        CHECK(transformable.getOrigin() == sf::Vector2f(0, 0));
+        transformable.setOrigin({7, 8});
+        CHECK(transformable.getOrigin() == sf::Vector2f(7, 8));
+
+        sf::Transform transform;
```

Comment:
Maybe from here onward should be a separate test case; not about setters/getters.

Reply:
I lumped them all together since `getTransform` and `getInverseTransform` are still `get` functions. If the transform getters became their own test case then I'd just copy all the set calls from this case and we'd end up with two significantly similar test cases.